examples/icd_code_prediction/config.yml

```diff
@@ -0,0 +1,11 @@
+Spacy:
+  processors: ["sentence", "tokenize", "pos", "ner", "umls_link"]
```

Comment:
If we only need tokenization from space, we need not add other pipes to the processors' list.